### ISSUE TITLE
GitHub Workshop: Add citation to "python" 

### ISSUE THREAD

**Issue by BrainonSilicon** [aim-rsf/Glossary-of-Terms#27]
# GitHub for Collaboration Workshop

This issue accompanies the GitHub for Collaboration Workshop developed by Eirini Zormpa, Sophia Batchelor and Rachael Stickland for the AIM RSF. We're so glad to have you here! 🎉 

Below you'll find some instructions for how to contribute to the AIM RSF Glossary of Terms and develop collaborative practices on GitHub. If you have any questions, please let @RayStick or @eirini-zormpa know on Zoom so that we can assist you.

## Summary
Terms that were identified and discussed during the September Collaboration Cafe need to be added to the Glossary of Terms.  

## Guide to Contribution 

You will be making 2 commits to a branch and then making a Pull Request that adds a citation to a term in the Glossary. Please follow the steps detailed below to complete this task. 

1. Go to the glossary file in this repository to address the issue: https://github.com/aim-rsf/Glossary-of-Terms/blob/main/docs/glossary.md

2. Click 'Edit this file' and find **line 49**  

3. Use Markdown format to add the following description at the end of the sentence 
```
{cite}`perez2011python`
```

4. Scroll down to the 'Propose changes' section at the bottom of the page: edit the title and the description of the issue to describe your update to the community.

5. Select "New Branch" and give your new branch a descriptive name.

6. Select the "Propose Changes" button. You will then be taken to a new screen that submits a Pull Request. Don't fill this out yet as we need to add the reference to the bibliography first. 

7. Navigate to the references.bib file which contains all the references according to BibTeX formatting: https://github.com/aim-rsf/Glossary-of-Terms/blob/main/docs/references.bib

8. On **line 13**, paste the following text. Please be careful and ensure that the white space has also been copied as the line indentation is really important. 
```
@article{perez2011python
,	title	= {Python: an ecosystem for scientific computing}
,	author	= {Perez, Fernando and Granger, Brian E and Hunter, John D}
,	journal	= {Computing in Science \\& Engineering}
,	volume	= {13}
,	number	= {2}
,	pages	= {13--21}
,	year	= {2011}
,	publisher	= {AIP Publishing}
}
```

9. Scroll down to the 'Propose changes' section at the bottom of the page: and add a descriptive title (for example: "Added python citation to bibtex file").

10. Check that you are committing your changes to the branch you created in the previous steps. Select the "Propose changes" button.

11. Navigate to the [Pull Requests page](https://github.com/aim-rsf/Glossary-of-Terms/pulls) and select "New Pull Request". 

12. Fill in any relevant information, including the issue that the pull request fixes. You can connect the pull request and the issue it addressed **using a`#`**. You can find the issue number next to the title of this issue (this one is 27).

For example: `this pull request fixes #27`

Including this sentence in the body of your pull request will automagically close the issue once the pull request is merged 🪄✨ 

13. Click 'Create Pull Request'! And Congratulations! 🎉 You're done!

14. Navigate to the  "Reviewers" section to the top righthand side of your Pull Request. Click on the cogwheel and select @eirini-zormpa, @RayStick, and @BrainonSilicon from the list. A reviewer will review your pull request, and ask you if you need to make any changes before it can be approved.

15. Once your pull request is approved and the checks are complete, you can merge your change into the repository! ✅

**Comment by BrainonSilicon** [comment]
Updated Glossary with Topic specific layout and addressed this issue in [this commit](746f8dc0550afcc38e1d75a26baa974d639cf1fe)
